modules/sql-engine/src/integrationTest/sql/sqlite/join/join1.test

```diff
@@ -22,12 +22,6 @@ INSERT INTO t2 VALUES(2,3,4);
 statement ok
 INSERT INTO t2 VALUES(3,4,5);
 
-query III rowsort
```

Comment:
let's better to use single test disable instead of moving to separate file
`skipif ignite3`

Reply:
done